### PR TITLE
Added commands for user: Add user, Execute transaction, Exit

### PR DESCRIPTION
**Pull request by 93artur**

### CODE REVIEW

File: src/main/java/com/arturobank/ArturoBank.java [hunk 1]
```diff
@@ -29,54 +29,99 @@ public String getCurrentDateTime() {
...
-            while (true) {
-                try {
-                    amount = Integer.parseInt(reader.readLine());
-                } catch (NumberFormatException e) {
-                    System.out.println("Use only Integers");
-                    continue;
+                if (command.equals("Add user")) {
+                    bank.addUser(reader);
+                } else {
+                    System.out.println("Select a command from the list");
```
Comment (IgorTsalko) [inline src/main/java/com/arturobank/ArturoBank.java:122]: If the user input does not match any command, I would suggest return warning message

File: src/main/java/com/arturobank/ArturoBank.java [hunk 2]
```diff
@@ -29,54 +29,99 @@ public String getCurrentDateTime() {
...
+        while (true) {
+            try {
+                amount = Integer.parseInt(reader.readLine());
+            } catch (NumberFormatException e) {
+                System.out.println("Use only Integers");
+                continue;
+            }
+            break;
+        }
+        this.sendToClient(sender, recipient, amount);
```
Comment (IgorTsalko) [inline src/main/java/com/arturobank/ArturoBank.java:94]: do you need "this" ?

Comment (93artur) [reply src/main/java/com/arturobank/ArturoBank.java:94]: resolved

File: src/main/java/com/arturobank/ArturoBank.java (hunk 1 above)
Comment (93artur) [reply src/main/java/com/arturobank/ArturoBank.java:122]: fixed

**Pull request merged**